source/s3_client.c

```diff
@@ -110,20 +111,25 @@ static void s_s3_client_on_acquire_http_connection(
     void *user_data);
 
 /* Schedule task for processing work. */
-static void s_s3_client_schedule_process_work_task_synced(struct aws_s3_client *client);
+static void s_s3_client_schedule_process_work_synced(struct aws_s3_client *client);
+
```

Comment:
What is the difference between these two functions? One is commented, one is not, and says default in it :)

Reply:
Default is the actual function in the vtable, the other one calls the vtable.  I'll add a comment for that!